tests/integration_tests/test_chainermn.py

```diff
@@ -419,14 +419,14 @@ def test_suggest_int(storage_mode: str, comm: CommunicatorBase, enable_log: bool
             for _ in range(10):
                 mn_trial = _create_new_chainermn_trial(study, comm)
 
-                x1 = mn_trial.suggest_int("x", low, high, step=step, log=enable_log)
+                x1 = mn_trial.suggest_int("y", low, high, step=step, log=False)
                 assert low <= x1 <= high
 
-                x2 = mn_trial.suggest_int("x", low, high, step=step, log=enable_log)
+                x2 = mn_trial.suggest_int("y", low, high, step=step, log=False)
```

Comment:
Why not use `y1`, `y2` as variable names now that you've changed the `name` argument? 
It seems like you don't have to parameterize (repeat) this logic any longer for `enable_log` but it's a minor detail.